universal-login-sdk/test/core/services/WalletService.unit.test.ts

```diff
@@ -76,7 +76,9 @@ describe('UNIT: WalletService', () => {
     } as any;
 
     storage = {
-      get: () => JSON.stringify({kind: 'Deployed', wallet: serializedDeployedWallet}),
+      get: () => {
+        return JSON.stringify({kind: 'DeployedWithoutEmail', wallet: serializedDeployedWithoutEmailWallet});
```

Comment:
Why don't we use here arrow function without `return`

Reply:
We should. I debugged this and forgot to return previous version